t/data/24-worker-overall/workers.ini

```diff
@@ -7,3 +7,7 @@ LOG_LEVEL = debug
 
 [1]
 WORKER_CLASS = qemu_i386,qemu_x86_64
+
+[2]
+WORKER_CLASS = qemu_i386,qemu_x86_64
+PARALLEL_ONE_HOST_ONLY = 1
```

Comment:
I suggest to not change this fixtures file but change the settings within the (sub-)test where you need it

Reply:
Done